src/components/ButtonPanel.js

```diff
@@ -1,34 +1,40 @@
 import React from 'react';
 import Button from './Button';
+import '../index.css';
 
-const ButtonPannel = () => {
-  const groups = [
-    ['AC', '+/-', '%', '÷'],
-    ['7', '8', '9', 'X'],
-    ['4', '5', '6', '-'],
-    ['1', '2', '3', '+'],
-    ['0', '.', '='],
-  ];
-
-  return (
-    <div className="App">
-      <div className="" id="G1">
-        { groups[0].map(e => <Button buttonName={e} key={e} />) }
-      </div>
-      <div className="" id="G2">
-        { groups[1].map(e => <Button buttonName={e} key={e} />) }
-      </div>
-      <div className="" id="G3">
-        { groups[2].map(e => <Button buttonName={e} key={e} />) }
-      </div>
-      <div className="" id="G4">
-        { groups[3].map(e => <Button buttonName={e} key={e} />) }
-      </div>
-      <div className="" id="G5">
-        { groups[4].map(e => <Button buttonName={e} key={e} />) }
-      </div>
+const ButtonPannel = () => (
+  <div className="App">
+    <div className="group" id="G1">
```

Comment:
**[Optional]**
You're not using `id` and anywhere in the code right?
And the `key` and `className` passed to the `Button` component here is not necessary, is not that you're using `map` right.